doc/sphinx/source/community/index.rst

```diff
@@ -9,6 +9,7 @@ Contributing to the community
 		Introduction <introduction>
 		Release schedule and procedure <release_strategy>
 		Contributing a diagnostic or recipe <diagnostic>
+        Contributing a dataset <dataset>
```

Comment:
the indentation looks wrong here

Reply:
Indentation with spaces instead of tabs...

Reply:
I guess we should match new additions to what is already there. I suggest to use tabs

Reply:
Yeah, my bad. My editor won't even allow me to indent with tabs, even when I copy-paste! 😅 Should be good now